food-delivery/apps/driver/next.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = () => {
+  const rewrites = () => {
+    return [
+      {
+        source: '/api/:path*',
+        destination: 'http://localhost:3000/api/:path*',
```

Comment:
Hmm... why redirect to localhost?

Reply:
It's a hack—redirecting to the other app's backend. Will be looking into configuring trpc client to just talk to other backend directly.